plugins/package-managers/build.gradle.kts

```diff
@@ -38,6 +38,7 @@ dependencies {
     api(project(":plugins:package-managers:gradle-inspector"))
     api(project(":plugins:package-managers:gradle-model"))
     api(project(":plugins:package-managers:gradle-package-manager"))
+    api(project(":plugins:package-managers:node-package-manager"))
```

Comment:
Commit message: Can you explain the reason why these package managers are moved into the same plugin and not into separate plugins?